roles/docker/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+# Default values for docker role
+
+docker_version: 1.9.1
```

Comment:
@mapuri looks like we might have to slightly more than bumping up the version.. issues that I have seen so far with 1.10

(1) ubuntu apt repository for 15.04 vivid does not have 1.10. So, ansible fails. Does not look like they want to support it. 15.10 has it. But, with 15.10 Vagrant is broken.
(2) centos72 - `start docker tcp service` fails.. because of `Feb 15 18:21:02 localhost docker: time="2016-02-15T18:21:02.453314126Z" level=fatal msg="Error starting daemon: error initializing graphdriver: \"/var/lib/docker\" contains other graphdrivers: overlay; Please cleanup or explicitly choose storage driver (-s <DRIVER>)"` - Not sure if it is because of 1.10 yet.